config/filter.d/qpsmtpd.conf

```diff
@@ -0,0 +1,11 @@
+
+[Definition]
+
+failregex = ^.*535 LOGIN authentication failed for .* from <HOST>$
```

Comment:
We'll don't accept pull requests with vulnerable REs anymore - not anchored (`^.*` is not an anchor) and several catch-all's (especially before `<HOST>`).

```diff
- failregex = ^.*535 LOGIN authentication failed for .* from <HOST>$
+ failregex = ^[^\[]+\[\d+\]: 535 LOGIN authentication failed for <F-USER>\S+|.*?(?= from)</F-USER> from <ADDR>$
```

Reply:
Thanks Sergey, I'll take a look at this and other changes, and resubmit.